website/pages/docs/job-specification/hcl2/index.mdx

```diff
@@ -0,0 +1,98 @@
+---
+layout: docs
+page_title: Configuration Language
+sidebar_title: HCL2 <sup>Beta</sup>
+description: |-
+  Noamd uses text files to describe infrastructure and to set variables.
+  These text files are called Nomad job specifications and are
+  written in the HCL language.
+---
+
+# HCL <sup>Beta</sup>
+
+`@include 'beta-nomad1.0-note.mdx'`
```

Comment:
This file is Nomad specific - it's the merge of Packer and existing Nomad doc in https://www.nomadproject.io/docs/job-specification/hcl2 .